css/src/components/gradient.scss

```diff
@@ -88,3 +88,32 @@ $border-gradient-direction: (
 	-webkit-text-fill-color: transparent;
 	line-height: 1.4; // required because of background clip, avoid "g" and "y" getting cut off
 }
+
+// Gradient card
```

Comment:
Appreciate the forward thinking here, but I think we can skip the loops here, in favor of making `vivid` the default. We can rename the component to just `gradient-card`. Down the line if we want more gradients we can add modifiers like `gradient-card-sunrise` for example.